paper/paper.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Altair: a declarative statistical visualization library for Python'
+tags:
+  - Python
+  - visualization
+  - statistics
+  - Jupyter
+authors:
+  - name: Jacob VanderPlas
+    orcid: 0000-0002-9623-3401
+    affiliation: 1
+  - name: Brian E. Granger
+    orcid: 0000-0002-5223-6168
+    affiliation: 2
+  - name: Jeffrey Heer
+    orcid: 0000-0002-6175-1655
+    affiliation: 1
+  - name: Eitan Lees
+    orcid: 0000-0003-0988-6015
+    affiliation: 4
+  - name: Dominik Moritz
+    orcid: 0000-0002-3110-1053
+    affiliation: 1
+  - name: Scott Sievert
+    orcid: 0000-0002-4275-3452
+    affiliation: 3
+  - name: Ben Welsh
+    orcid: 0000-0002-5200-7269
+    affiliation: 5
+  - name: Kanit Wongsuphasawat
+    orcid: 0000-0001-7231-279X
+    affiliation: 1
+affiliations:
+  - name: University of Washington
+    index: 1
+  - name: California Polytechnic State University, San Luis Obispo
+    index: 2
+  - name: University of Wisconsin, Madison
+    index: 3
+  - name: Florida State University
+    index: 4
+  - name: Los Angeles Times Data Desk
+    index: 5
+date: 01 June 2018
+bibliography: paper.bib
+---
+
+# Summary
+
+Altair is a statistical visualization library for Python. Statistical visualization is a constrained subset of data visualization focused on the creation of visualizations
+that are helpful in statistical modeling. The constrained model of statistical visualization is usually expressed in terms of a visualization grammar [] that specifies how input data is transformed and mapped to visual properties (position, color, size, etc.).
+
+Altair is based on the Vega-Lite visualization grammar [], which allows a wide range of statistical
+visualizations to be expressed using a small number of grammar primitives. Vega-Lite is declarative; visualizations are specified using JSON data that follows the Vega-Lite JSON schema.
+As a Python library, Altair provides an API oriented towards scientists and data scientists
+doing exploratory data analysis []. Altair's Python API emits Vega-Lite JSON data, which is then
+rendered in a user-interface such as the Jupyter Notebook, JupyterLab or nteract using the
+Vega-Lite JavaScript library []. Vega-Lite JSON is compiled to a full Vega specification [], which is then parsed and executed using a reactive runtime that internally makes use of D3.js [].
+
+The declarative nature of the Vega-Lite visualization grammar, and its encoding in a formal
+JSON schema, provide Altair with a number of benefits. First, much of the Altair Python code,
+tests, and examples are autogenerated from the Vega-Lite JSON schema, ensuring strict conformance
```

Comment:
remove "examples" here, because we no longer auto-generate them

Reply:
Done in https://github.com/ellisonbg/altair/pull/4/commits/6e995d0a515cfc6751ab83b9181f142a713d86d6